graphrag_eval/aggregation.py

```diff
@@ -131,9 +131,10 @@ def compute_per_template_stats(
 
 
 def compute_micro_stats(
-    number_of_samples_per_template_by_status,
-    stats_per_template,
-    step_metrics_per_template
+    number_of_samples_per_template_by_status: dict[str, dict[str, int]],
+    stats_per_template: dict[str, dict[str, Sequence[int]]],
+    steps_summary_per_template: dict[str, dict[str, dict[str, int]]],
+    step_metrics_per_template: dict[str, dict[str, Sequence[int]]],
```

Comment:
Is this used?

Reply:
once_per_sample: how many times each step was executed, counted only once per question